### DISCUSSION TITLE
Add album names for multiple languages; Show album name depending on client's Accept-Language HTTP header

### DISCUSSION THREAD

**Discussion by schnippl0r** [LycheeOrg/Lychee#2559] [Ideas]
### Enhancement

Give an album owner the possibility to specify album titles for several languages (including one fallback/default language) and then let Lychee decide via the client's HTTP header `Accept-Language` which language for the album title is chosen for display.

Imagine the following situation:

I have an album with photos of the city Geneva. For English users (or users without the `Accept-Language` HTTP header), I want the album title to be Geneva, for French users it should be Genève, for German users it should be Genf, for Spanish users it should be Ginebra and so on.

So I want to be able to specify an arbitrary list of (language, album name) pairs.

For any client browsing albums, Lychee then should decide depending on the client's HTTP `Accept-Header` which album name to choose.  If no `Accept-Header` is set, or the language specified in the header is not an element of the language mapping that the album owner has set, the default album name should be displayed.

Album owners that do not wish to use this functionality, should not be bothered with a language mapping. A language mapping should only take place if the album owner clicks on a button like "Add translation" and then the currently entered album name should be mapped to the fallback/default language with the option to add another language.

Mockup:
![mockup1](https://github.com/user-attachments/assets/910bc040-aa50-448b-a809-2da61e695560)

![mockup2](https://github.com/user-attachments/assets/4a066b46-5ff3-4818-8162-82ce106087a2)

**Comment by ildyria** [comment]
That is way more complex that you make it sound like. :')
And I have a lot of other translation to take care of before I can even consider multi language translations.

To be brutally honest, I think this is more of the "nice to have" and there are a few things which are more in the "more important to have": e.g. user groups.

I will leave this as project for a volunteer.
